### PR TITLE
add adfs_authority_url arg in azure_rm.py

### PR DESCRIPTION
**Pull request by yungezz**
##### SUMMARY

fix issue #40824, add adfs_authority_url to argument list.

##### ISSUE TYPE

 - Bugfix Pull Request

##### ANSIBLE VERSION

```
ansible 2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by yungezz** [comment]
@yuwzho

File: contrib/inventory/azure_rm.py [hunk 1]
```diff
@@ -326,10 +326,10 @@ def __init__(self, args):
         # get authentication authority
         # for adfs, user could pass in authority or not.
         # for others, use default authority from cloud environment
-        if self.credentials.get('adfs_authority_url') is None:
```
Comment (yuwzho) [inline contrib/inventory/azure_rm.py:329]: What's the difference?

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

File: contrib/inventory/azure_rm.py [hunk 2]
```diff
@@ -326,10 +326,10 @@ def __init__(self, args):
...
-            self._adfs_authority_url = self._cloud_environment.endpoints.active_directory
-        else:
+        if self.credentials.get('adfs_authority_url'):
             self._adfs_authority_url = self.credentials.get('adfs_authority_url')
```
Comment (zikalino) [inline contrib/inventory/azure_rm.py:330]: i would do:

self._adfs_authority_url = self.credentials.get('adfs_authority_url') 
if self._adfs_authority_url is None:
   ...

**Review by zikalino — APPROVED** [review]

**Pull request merged**